lib/contracts/code_templates/main-context.js.ejs

```diff
@@ -1 +1,5 @@
-var __mainContext = __mainContext || this;
+var __mainContext = __mainContext || (
+    typeof this !== 'undefined' ? this : (
+        typeof self !== 'undefined' ? self : void 0
```

Comment:
What is `void 0`?

Reply:
`void 0` is a way of writing the value `undefined` (not the string `'undefined'`), and will work even if someone redefined the global value of `undefined`, which JS allows.